ecs-cluster/alerts/remedies/12001_UNREAD_BYTES_DURING_PAUSE_issue.yaml

```
@@ -1,6 +1,6 @@
 - symptomid: OBJST-12001
   description: Replication stream has large backlog in pravega.
-  component: crr
+  component: Geo replication
```

```suggestion
  component: "Geo replication"
```
probably needs to be quoted.  right @nikolay-t  @vgupta-mickey

---

ok，thanks